opentasks/src/main/res/values-cy/strings.xml

```diff
@@ -0,0 +1,293 @@
+
+    <!-- Strings for QuickSettingsTile -->
+    <string name="quicksettingstile_create">Creu Tasg</string>
+
+    <!-- Dashclock -->
+    <string name="dashclock_extension_title">Tasgau</string>
+    <string name="dashclock_extension_description">Mae'n dangos gwybodaeth am dasgau diweddar.</string>
+    <string name="dashclock_widget_title_due_expanded">Rhaid cyflawni %1$s - %2$s</string>
+    <string name="dashclock_widget_title_due_expanded_allday">Rhaid cyflawni %1$s</string>
+    <string name="dashclock_widget_title_start_expanded">Mae %1$s yn dechrau - %2$s</string>
+    <string name="dashclock_widget_title_start_expanded_allday">Mae %1$s yn dechrau heddiw</string>
+    <string name="dashclock_pref_displayed_task_title">Digwyddiadau a ddangosir</string>
+    <string name="dashclock_pref_displayed_task_summary">Dewisa pa ddigwyddiadau tasgau i'w dangos yn y teclyn DashGloc.</string>
+    <string name="dashclock_pref_displayed_task_dialog_title">Digwyddiadau i'w dangos</string>
+    <string name="dashclock_display_type_due">tasgau yn derfyn</string>
+    <string name="dashclock_display_type_start">tasgau yn dechrau</string>
+    <string name="dashclock_display_type_start_and_due">tasgau yn dechrau a therfyn</string>
+    <string name="dashclock_display_type_all">y cwbl</string>
+    <string name="dashclock_display_type_pinned">tasgau a hoeliwyd</string>
+    <string name="dashclock_preference_title">Task DashClock Preferences</string>
+
+    <!-- General -->
+    <string name="accept">Derbyn</string>
+    <string name="now">nawr</string>
+
+    <!-- Incoming share -->
+    <string name="share_as_task">Creu Tasg</string>
+
+    <!-- Settings -->
+    <!-- Notification enable/disable switch label in settings -->
+    <string name="opentasks_settings_notification_enable">Hysbysiadau</string>
+    <!-- Section title in settings for the notification signals -->
+    <string name="opentasks_settings_notification_signals_section_title">Dulliau Hysbysu</string>
+    <!-- Notification Sound enabling/disabling entry label in settings -->
+    <string name="opentasks_settings_notification_sound">Sain</string>
+    <!-- Notification Vibration enabling/disabling entry label in settings -->
+    <string name="opentasks_settings_notification_vibrate">Dirgrynu</string>
+    <!-- Notification "Pulse light" enabling/disabling entry label in settings -->
+    <string name="opentasks_settings_notification_lights">Golau curiad</string>
+
+    <!-- Outgoing share -->
+    <string name="opentasks_share_footer">Rhannwyd gydag OpenTasks</string>
+
+    <!-- GET_ACCOUNTS permission (in CONTACTS permission group) requesting dialog: -->
+    <string name="opentasks_permission_request_dialog_getaccounts_title">Angen caniatâd</string>
+    <string name="opentasks_permission_request_dialog_getaccounts_message"><![CDATA[In order to read the list of accounts on your device which support task synchronization, %s needs the GET_ACCOUNTS permission.<br/><br/>
```

Comment:
You can just remove untranslated strings. That way its easier to spot that this is still untranslated.

Reply:
Translated.